includes/admin/onboarding/onboarding.php

```diff
@@ -0,0 +1,596 @@
+			}
+		}
+	}
+
+	/**
+	 * Set current onboarding step.
+	 *
+	 * @since 3.2
+	 */
+	public function set_current_onboarding_step() {
+		// If Onboarding hasn't started yet, we force the first default step.
+		if ( ! $this->onboarding_started ) {
+			return;
+		}
+
+		// User is requesting a specific step.
+		if( isset( $_GET['current_step'] ) ) {
+			$this->current_step = sanitize_key( $_GET['current_step'] );
+		} else {
+			$this->current_step = sanitize_key( get_option( 'edd_onboarding_latest_step', $this->current_step ) );
+		}
+
+		// If requested step does not exist, abort.
+		if ( ! isset( $this->onboarding_steps[ $this->current_step ] ) ) {
+			wp_die( __( 'Unknown Onboarding Step.', 'easy-digital-downloads' ), __( 'Onboarding Wizard', 'easy-digital-downloads' ), 404 );
+		}
+
+		$this->current_step_index = $this->onboarding_steps[ $this->current_step ]['step_index'];
+		update_option( 'edd_onboarding_latest_step', $this->current_step, false );
+	}
+
+	/**
+	 * Get previous step.
+	 *
+	 * @since 3.2
+	 */
+	public function get_previous_step() {
+		$internal_step = $this->current_step_index - 2;
+		$step_keys     = array_keys( $this->onboarding_steps );
+		if ( isset( $step_keys[ $internal_step ] ) ) {
+			return $step_keys[ $internal_step ];
+		}
+
+		return false;
+	}
+
+	/**
+	 * Get current step.
+	 *
+	 * @since 3.2
+	 */
+	public function get_current_step() {
+		return $this->current_step;
+	}
+
+	/**
+	 * Get current step details.
+	 *
+	 * @since 3.2
+	 */
+	public function get_current_step_details() {
+		return $this->onboarding_steps[ $this->get_current_step() ];
+	}
+
+	/**
+	 * Get next step.
+	 *
+	 * @since 3.2
+	 */
+	public function get_next_step() {
+		$internal_step = $this->current_step_index;
+		$step_keys     = array_keys( $this->onboarding_steps );
+		if ( isset( $step_keys[ $internal_step ] ) ) {
+			return $step_keys[ $internal_step ];
+		}
+
+		return false;
+	}
+
+	/**
+	 * Get pagination.
+	 *
+	 * @since 3.2
+	 */
+	public function get_step_pagination() {
+		return array(
+			'previous' => $this->get_previous_step(),
+			'current'  => $this->get_current_step(),
+			'next'     => $this->get_next_step(),
+		);
+	}
+
+	/**
+	 * Onboarding Wizard subpage screen.
+	 *
+	 * @since 3.2
+	 */
+	public function onboarding_wizard_sub_page() {
+		$onboarding_initial_style = ( ! $this->onboarding_started ) ? ' style="display:none;"' : '';
+		?>
+		<?php wp_nonce_field( 'edd_onboarding_wizard' ); ?>
+		<div class="edd-onboarding">
+			<div class="edd-onboarding__logo">
+				<img src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/logo-edd-dark.svg' ); ?>">
+			</div>
+			<div class="edd-onboarding__wrapper">
+				<div class="edd-onboarding__loading" style="display: none;">
+					<div class="edd-onboarding__loading-content-wrapper">
+						<img src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/oval-ajax-loader.svg' ); ?>" alt="">
+						<div class="edd-onboarding__loading-status"></div>
```

Comment:
When this displays, I can't actually read it:
![image](https://user-images.githubusercontent.com/4853066/201956897-106af037-1cb2-4699-b96b-0a3e1e0616cc.png)